rules_python_zip/simplepack.go

```diff
@@ -501,6 +502,14 @@ if need_unzip and isinstance(__loader__, zipimport.zipimporter):
     tempdir_create_pid = os.getpid()
     atexit.register(clean_tempdir_parent_only, tempdir)
     sys.path.insert(0, tempdir)
+    # Handle linux signal terminate by calling exit, so atexit code executes.
```

Comment:
If we do end up putting this in, can you please send an equivalent PR to the main pyz repo? I don't want to diverge any further, given that we will very likely want to merge in upstream changes in the near future.